ansible/roles/ami-ovf/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+    pmm2_server_release_repo: release
```

Comment:
Variable name should not contain its value :) A better name would be `pmm2_server_repo_final` or something like that.

Reply:
fixed